### DISCUSSION TITLE
I didn't receive my benefits after 5 days.

### DISCUSSION THREAD

**Discussion by gnurab** [community/community#146354] [GitHub Education]
### Hi everyone,

Hi everyone, 

I submit my application for the Github Student Developer Pack, i waited around 5 days and still didn't receive my benefits.

I would be glad for some help.

Thank you.

**Comment by cachedcake** [comment]
Hi @gnurab,

Don't worry it's normal, when the benefits become available, the status label will turn purple and change to:

- Congrats! Your request for @gnurab was approved and your benefits are now available.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions:
[#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470), [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279), [#135849](https://github.com/orgs/community/discussions/135849), [#136887](https://github.com/orgs/community/discussions/136887), [#138721](https://github.com/orgs/community/discussions/138721), [#139268](https://github.com/orgs/community/discussions/139268), [#140287](https://github.com/orgs/community/discussions/140287), [#145640](https://github.com/orgs/community/discussions/145640)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Let me know of this helps! (Written by @davevad93)
